package.json

```diff
@@ -5,12 +5,11 @@
   "description": "Client browser fingerprinting mechanism",
   "main": "index.js",
   "scripts": {
-    "test": "nyc ava",
-    "test:watch": "ava -vw",
-    "build": "webpack",
-    "dev": "webpack-dev-server",
-    "lint": "eslint --fix src/**/*.js",
-    "lint:staged": "lint-staged"
```

Comment:
use "precommit" and update `husky` config

Reply:
Example repo to demonstrate use of `lint-staged` with multi-pkg projects - https://github.com/sudo-suhas/lint-staged-multi-pkg